cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -1,61 +1,146 @@
+{% import '_vars-mega-menu.html' as vars with context %}
+{% set base_class = 'o-mega-menu' %}
 
 {# ==========================================================================
 
-   sub_nav()
+   _classes()
```

Comment:
We might want to pull this out in a future PR and reuse it in the sec-nav so that we're consistent across navs. Should we add a note so we don't forget?